{{ cookiecutter.__package_name_kebab_case }}/pyproject.toml

```diff
@@ -169,17 +169,70 @@ target-version = "py{{ cookiecutter.python_version|replace('.', '') }}"
 [tool.poe.tasks]  # https://github.com/nat-n/poethepoet
 {%- if cookiecutter.with_fastapi_api|int %}
 
-  [tool.poe.tasks.dev]
-  help = "Serve a REST API in development"
+  [tool.poe.tasks.api]
+  help = "Run a REST API"
+  shell = """
+    if [ $dev ]
+    then {
+      uvicorn \
+        --host $host \
+        --port $port \
+        --reload \
+        minifigures_api.api:app
```

Comment:
```suggestion
        {{ cookiecutter.__package_name_snake_case }}.api:app
```

Reply:
Yes I did try out the flag. 
Thank you for fixing the project name 😓